how-we-work.md

```diff
@@ -14,7 +14,7 @@ Then there are ad-hoc in-person meetups that happen about once or twice a month,
 
 We work in 2-week Scrum Sprints described in detail in [Work Process](https://github.com/niteoweb/handbook/blob/master/work-process.md). 
 
-Project and task management is currently done with [GitHub](https://github.com/). Support handles tickets through [GrooveHQ](https://www.groovehq.com). We also have an internal document system (intranet) we call “intra”, running on [Plone](https://plone.org/), where all our company processes and documents are stored.
+Project and task management is currently done with [GitHub](https://github.com/) and [ZenHub](https://www.zenhub.com/) for better visibility and predictability. Support handles tickets through [GrooveHQ](https://www.groovehq.com). We also have an internal document system (intranet) we call “intra”, running on [Plone](https://plone.org/), where all our company processes and documents are stored.
```

Comment:
Why not? Is there any other reason why we use it? Or do you think we should just remove this part because it is obvious why we are using it?